packages/oauth/oauth-client/src/oauth-client.ts

```diff
@@ -297,6 +347,13 @@ export class OAuthClient {
     )
   }
 
+  async abortRequest(authorizeUrl: URL) {
+    const requestUri = authorizeUrl.searchParams.get('request_uri')
+    if (!requestUri) return
+
+    // @TODO find a way to actually do this
```

Comment:
Just flagging to ensure this TODO is still intended to be here.

Reply:
This is fine for now. Improved my comments there.